### DISCUSSION TITLE
Certificate Error Cubbit point

### DISCUSSION THREAD

**Discussion by Sorgaz** [iterate-ch/cyberduck#15473] [Q&A]
Hi
I recently purchased a Mountain Duck license to be able to connect to Cloud Object Personal Storage of the Cubbit  https://docs.cubbit.io/integrations/mountainduck.

Following the installation and setting as per guide, Mountain Duck presents the same error in an error error (see attached image) even if I accept and install the certain the pop-up it comes out persistently not allowing synchronization.

Someone have idea?
![MountainDuck](https://github.com/iterate-ch/cyberduck/assets/45358309/41d9c2f6-47f7-41ce-89e4-d89d25f44009)

**Comment by dkocher** [comment]
The server `cubbit.di.shalashaska.s3.cubbit.eu` advertises a certificate with a common name (CN) `Kubernetes Ingress Controller Fake Certificate` which is not signed by a well known certificate authority that is trusted in Windows. Please reach out to _Cubbit_ support about this configuration error in their system.
